LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
```

Comment:
I don't think we can just move this file. There's a number of directories without an explicit LICENSE file right now, my sense is that our best bet would be to do a checkbox issue to get approvals and then land this PR, similar to those done for many other rust-lang repos (e.g., https://github.com/rust-lang/rfcs/issues/2076).

Reply:
Done in https://github.com/rust-lang/rustc-perf/issues/1933.